### PR TITLE
Add `strwch` option to autostart system watch

### PR DESCRIPTION
**Pull request by ThePrez**
Introduces a `strwch` option to `data.ini`'s watch event which allows for auto starting/stopping of the watch

Example configuration in `data.ini`:

```ini
[watchout]
type=watch
id=jesse
destinations=test_out, slackme
format=$MESSAGE_ID$ (severity $SEVERITY$): $MESSAGE$ 
strwch=STRWCH SSNID(ZZZ) WCHPGM(MANZAN/HANDLER) CALLWCHPGM(*STRWCH) WCHMSG((*ALL)) WCHMSGQ((*HSTLOG))
```

Of note: the `STRWCH` command name, as well as the `SSNID` and `WCHPGM` arguments are optional. All will be replaced with appropriate values as needed. For instance, the above configuration could more concisely be written as:

```ini
[watchout]
type=watch
id=jesse
destinations=test_out, slackme
format=$MESSAGE_ID$ (severity $SEVERITY$): $MESSAGE$ 
strwch=WCHMSG((*ALL)) WCHMSGQ((*HSTLOG))
```

### CODE REVIEW

File: camel/src/main/java/com/github/theprez/manzan/WatchStarter.java [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    public void strwch()
+            throws IOException, AS400SecurityException, ErrorCompletingRequestException, InterruptedException,
+            PropertyVetoException, ObjectDoesNotExistException {
+        if (isRunning()) {
+            System.err.println("Watch '" + m_session_id + "' already running");
+            return;
+        }
+        runCmd(m_command);
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
```
Comment (worksofliam) [inline camel/src/main/java/com/github/theprez/manzan/WatchStarter.java:57]: Will this be invoked even if the process is killed, or if interrupted with Control+C?

**Review by worksofliam — APPROVED** [review]
Looks good. Please check out my question.

File: camel/src/main/java/com/github/theprez/manzan/WatchStarter.java (hunk 1 above)
Comment (ThePrez) [reply camel/src/main/java/com/github/theprez/manzan/WatchStarter.java:57]: > Will this be invoked even if the process is killed, or if interrupted with Control+C? 

Yep. ENDJOb too. 

It won't run in cases of immediate job/vm termination, though, like:
- stopping from the debugger
- JVM crash

Comment (worksofliam) [reply camel/src/main/java/com/github/theprez/manzan/WatchStarter.java:57]: I am ok with that. Thanks!

**Pull request merged**